common/js/src/popup-window/in-popup-main-script.js

```diff
@@ -99,7 +100,8 @@ GSC.InPopupMainScript.prepareAndShowAsModalDialog = function() {
   GSC.InPopupMainScript.setWindowHeightToFitContent();
   GSC.InPopupMainScript.setupClosingOnEscape();
   GSC.InPopupMainScript.setupRejectionOnWindowClose();
-  GSC.InPopupMainScript.showWindow();
+  if (GSC.Packaging.MODE == GSC.Packaging.Mode.APP)
```

Comment:
nit: Triple-equality (the identity operator) is preferred. See https://google.github.io/styleguide/jsguide.html#features-equality-checks 

P.S. I used the double-equality in my PR by mistake.